src/components/App/App.css

```diff
@@ -1,5 +1,38 @@
-.name{
-  font-size: 2em;
-  height: 5em;
-  background-color: red;
-}
\ No newline at end of file
+.App {
```

Comment:
I already removed this and your PR is bringing it back. Not a big deal but something to keep an eye on.